conda/recipes/rapids-build-env/meta.yaml

```diff
@@ -51,7 +51,7 @@ requirements:
     - conda-forge::clang {{ clang_version }} # [x86_64]
     - conda-forge::clang-tools {{ clang_version }} # [x86_64]
     - cmake {{ cmake_version }}
-    - cmake-format {{ cmake_format_version }}
+    - cmakelang {{ cmake_format_version }}
```

Comment:
@vyasr, was this intentionally changed to `cmakelang`? It looks like the CI logs are showing the error message below after this change. It doesn't seem like that package exists.

![image](https://user-images.githubusercontent.com/7400326/158600887-3a423bd7-8a64-43ba-986e-3cd81ca6bfc7.png)